apps/backend/src/routes/signup/+page.svelte

```diff
@@ -0,0 +1,243 @@
+      if (data.password === data.confirmPassword) {
+        // This will only be executed if passwords match
+        await signUpEmailPassword(data.email, data.password, (error) => (apiError = error));
+        console.log('Account created successfully');
+      } else {
+        console.error('Passwords do not match');
+      }
+    },
+  });
+</script>
+
+<svelte:head>
+  <title>Sign Up</title>
+</svelte:head>
+
+<!-- Armadillo Logo-->
+<div class="flex items-center justify-center p-10">
+  <img src="armadillo.png" alt="" />
+</div>
+
+<div class="flex flex-row items-center justify-center">
+  <!-- Signup Form - Card -->
+  <div class="w-full max-w-2xl p-10 border border-secondary rounded-md shadow-lg">
+    <!-- Signup Form - Header -->
+    <h1 class="text-3xl font-semibold text-center mb-5">Create your account</h1>
+
+    <form on:submit|preventDefault={handleSubmit}>
+      <!-- General Error Message -->
+      {#if apiError && apiError.code === 'auth/email-already-in-use'}
+        <div class="p-4 flex flex-row bg-red-50 text-red-800 rounded-lg" role="alert">
+          <img src="errorlogo.svg" alt="" />
+          <div class="ml-2 text-sm font-medium"
+            >You're too fast, slow down and try again in a while</div>
+        </div>
+      {/if}
+
+      <!-- Signup Form - Corporate Email Input -->
+      <div>
+        <label for="email" class="form-control w-full max-w-2xl">
+          <div class="label">
+            <span class="label-text">Corporate Email Address</span>
+            <span class="text-red-600 label-text-alt">*</span>
+          </div>
+          <input
+            id="email"
+            type="text"
+            name="email"
+            placeholder="Name@example.com"
+            on:change={handleChange}
+            bind:value={$form.email}
+            class="
+              input
+              input-bordered
+              w-full max-w-2xl
+              focus:ring-secondary
+              focus:border-secondary
+              duration-300
+              focus:outline-none
+            " />
+        </label>
+        <div>
+          <span class="text-red-600">{$errors.email}</span>
+        </div>
+      </div>
+
+      <!-- Signup Form - Password Input -->
+      <div>
+        <label for="password" class="form-control w-full max-w-2xl">
+          <div class="label">
+            <span class="label-text">Password</span>
+            <span class="text-red-600 label-text-alt">*</span>
+          </div>
+          <input
+            id="password"
+            type="password"
+            name="password"
+            placeholder="••••••••"
+            on:change={handleChange}
+            bind:value={$form.password}
+            class="
+              input
+              input-bordered
+              w-full
+              max-w-2xl
+              focus:ring-secondary
+              focus:border-secondary
+              duration-300
+              focus:outline-none
+            " />
```

Comment:
When I say multiline, I don't mean this.

E.g.,
```
w-full max-w-md input input-bordered duration-300
focus:outline-none focus:ring-secondary focus:border-secondary
```

Reply:
Changed accordinly